task/kubernetes-actions/0.2/README.md

```diff
@@ -0,0 +1,68 @@
+# kubernetes actions
+
+This is a generic task used to perform kubernetes actions such as `kubectl get deployment` or `kubectl create -f filename.yaml`. For more commands [see](https://kubernetes.io/docs/reference/kubectl/overview/).
+
+## Install the task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/kubernetes-actions/0.2/kubernetes-actions.yaml
+```
+
+## Inputs
+
+### Parameters
+
+- **script**: script of `kubectl` commands to execute e.g. `kubectl get pod $1 -0 yaml`. This will take the first value of ARGS as pod name (_default_: `kubectl $@`)
+- **args**: args to execute which are appended to `kubectl` e.g. `start-build myapp` (_default_: `help`)
+- **image**: Default image being `gcr.io/cloud-builders/kubectl`. If somebody wants to use their own image then they can provide it as a part of params. For example an image avilable is `lachlanevenson/k8s-kubectl`
+
+### Workspaces(Optional)
+
+- **kubeconfig-dir**: If you want to deploy you application to another cluster then you can mount your `kubeconfig` file via this `workspace`.
+- **manifest-dir**: Manifest files can be provided via the workspaces.
+
+## Results
+
+- **output-result**: If you want to emit some result which can be used in decision making. One such case is when using `when` expressions.
```

Comment:
It will be good if we can add hyperlink to the documentation of `when`, as this is pretty new feature in the pipeline (some users may not know this)

Reply:
addressed :+1: